keel-sql/src/main/resources/db/changelog/20191108-environment-artifact-constraint.yml

```diff
@@ -75,3 +75,11 @@ databaseChangeLog:
             columns:
               - column:
                   name: judged_at
+
+  - changeSet:
+      id: mysql-environment-artifact-constraint-status-not-enum
+      author: asher
+      changes:
+        - sql:
+            dbms: mysql
+            sql: ALTER TABLE `environment_artifact_constraint` MODIFY COLUMN `status` VARCHAR(64)
```

Comment:
Jooq not properly supporting enums should have been a signal that jooq auto-class generation wouldn't support them either.

Reply:
What was the column type previously? We should never store enums by ordinal!

Reply:
It was originally a native sql ENUM.